controllers/ixiatg_controller.go

```diff
@@ -1204,20 +1228,21 @@ func (r *IxiaTGReconciler) containersForIxia(podName string, intfList []string,
 		}
 		if cName == IMAGE_PROTOCOL_ENG {
 			compCopy.DefEnv["INTF_LIST"] = strings.Join(intfList, ",")
-			if compCopy.LiveNessEnable == nil || *compCopy.LiveNessEnable {
-				tcpSock := corev1.TCPSocketAction{Port: intstr.IntOrString{IntVal: 50071}}
-				pbHdlr := corev1.ProbeHandler{TCPSocket: &tcpSock}
-				probe := corev1.Probe{
-					ProbeHandler:                  pbHdlr,
-					InitialDelaySeconds:           compCopy.LiveNessDelay,
-					PeriodSeconds:                 compCopy.LiveNessPeriod,
-					FailureThreshold:              compCopy.LiveNessFailure,
-					TerminationGracePeriodSeconds: pointer.Int64(1),
-				}
-				container.LivenessProbe = &probe
-			}
+			tcpSock = corev1.TCPSocketAction{Port: intstr.IntOrString{IntVal: PROTOCOL_ENG_PORT}}
```

Comment:
To be addressed later: Currently we're manually adding probes for each and every ports that are being exposed. This is problematic as in future protocol engine will expose more ports (i.e. for CPDP communication).

It's just an example, but I believe we can refactor prob addition code so that probe gets added regardless of TCP port or corresponding container.

Reply:
To be taken up in later release.